lib/code_corps/user_task_matcher.ex

```diff
@@ -0,0 +1,25 @@
+defmodule CodeCorps.UserTaskMatcher do
+  @moduledoc """
+  Find the top tasks most matching a User's skills
+  """
+
+  alias CodeCorps.{Repo, Task, User, TaskSkill}
+  import Ecto.Query
+
+  @spec match_user(User.t, number) :: [Task.t]
+  def match_user(%CodeCorps.User{} = user, tasks_count) do
```

Comment:
Would write some inline documentation here and rename to `tasks_limit` or something more intention revealing.